lib/active_merchant/billing/gateways/securion_pay.rb

```diff
@@ -92,8 +115,18 @@ def scrub(transcript)
 
       private
 
+      def create_customer_add_card(authorization, options)
+        post = {}
+        post[:email] = options[:email]
+        post[:description] = options[:description]
+        post[:card] = authorization
+        post[:metadata] = {}
+        post[:metadata][:chargeId] = authorization
+        commit('customers', post, options)
+      end
+
       def add_customer(post, payment, options)
-        post[:customer] = options[:customer] if options[:customer]
+        post[:customerId] = options[:customer_id] if options[:customer_id]
```

Comment:
As far as I checked, :customer was only being populated in the adapter's auth/purchase methods. Both those methods send out POST requests to create a new Charge. The create new charge documentation lists :customerId as a data input (and not :customer). Also, :customer has never been exercised in any of the existing tests. All this lead to believe :customer has either been unused or erroneous.